tests/acceptance/features/bootstrap/SharingNgContext.php

```diff
@@ -143,6 +143,7 @@ public function userCreatesAPublicLinkShareWithSettings(string $user, TableNode
 
 		$bodyRows['displayName'] = \array_key_exists('displayName', $bodyRows) ? $bodyRows['displayName'] : null;
 		$bodyRows['expirationDateTime'] = \array_key_exists('expirationDateTime', $bodyRows) ? $bodyRows['expirationDateTime'] : null;
+		$bodyRows['password'] = $bodyRows['password'] ?? null;
```

Comment:
use the short form of ternary op
```suggestion
		$bodyRows['displayName'] = $bodyRows['displayName'] ?: null;
		$bodyRows['expirationDateTime'] = $bodyRows['expirationDateTime'] ?: null;
		$bodyRows['password'] = $bodyRows['password'] ?: null;
```